frontend/app/views/support/ABTest.scala

```diff
@@ -6,44 +6,77 @@ import java.net.URLEncoder
+  }
 }
 
 object AmountHighlightTest extends TestTrait {
 
   def name = "AmountHighlightTest"
   def slug = "highlight"
   override type VariantFn = (CountryGroup, Option[Int]) => Html
-
+  private lazy val notAustralia: Set[CountryGroup] = Set(
+    UK,
+    US,
+    Canada,
+    NewZealand,
+    Europe,
+    RestOfTheWorld
+  )
   def variants = NonEmptyList(
-    Variant("Amount - 5 highlight","5",0,views.html.fragments.giraffe.contributeAmountButtons(List(5,25,50,100),Some(5))),
-    Variant("Amount - 25 highlight","25",0,views.html.fragments.giraffe.contributeAmountButtons(List(5,25,50,100),Some(25))),
-    Variant("Amount - no highlight","None",0,views.html.fragments.giraffe.contributeAmountButtons(List(5,25,50,100), None)),
-    Variant("Amount - 35 highlight","35",0,views.html.fragments.giraffe.contributeAmountButtons(List(10,35,65,100),Some(35))),
-    Variant("Amount - 35 highlight descending","35-descending",0,views.html.fragments.giraffe.contributeAmountButtons(List(100,65,35,10),Some(35))),
-    Variant("Amount - 100 highlight","100",0,views.html.fragments.giraffe.contributeAmountButtons(List(25,50,100,250),Some(100))),
-    Variant("Amount - 50 highlight","50",1,views.html.fragments.giraffe.contributeAmountButtons(List(25,50,100,250),Some(50))),
-    Variant("Amount - 15","15",0,views.html.fragments.giraffe.contributeAmountButtons(List(15,35,65,100),Some(35))),
-    Variant("Amount - 40 highlight","40",0,views.html.fragments.giraffe.contributeAmountButtons(List(20,40,75,100),Some(40)))
+    Variant("Amount - 5 highlight","5",0,views.html.fragments.giraffe.contributeAmountButtons(List(5,25,50,100),Some(5)),notAustralia),
+    Variant("Amount - 25 highlight","25",0,views.html.fragments.giraffe.contributeAmountButtons(List(5,25,50,100),Some(25)), notAustralia),
+    Variant("Amount - no highlight","None",0,views.html.fragments.giraffe.contributeAmountButtons(List(5,25,50,100), None), notAustralia),
+    Variant("Amount - 35 highlight","35",0,views.html.fragments.giraffe.contributeAmountButtons(List(10,35,65,100),Some(35)), notAustralia),
+    Variant("Amount - 35 highlight descending","35-descending",0,views.html.fragments.giraffe.contributeAmountButtons(List(100,65,35,10),Some(35)), notAustralia),
+    Variant("Amount - 100 highlight","100",0,views.html.fragments.giraffe.contributeAmountButtons(List(25,50,100,250),Some(100)), notAustralia),
+    Variant("Amount - 50 highlight","50",1,views.html.fragments.giraffe.contributeAmountButtons(List(25,50,100,250),Some(50)), notAustralia),
+    Variant("Amount - 15","15",0,views.html.fragments.giraffe.contributeAmountButtons(List(15,35,65,100),Some(35)), notAustralia),
+    Variant("Amount - 40 highlight","40",0,views.html.fragments.giraffe.contributeAmountButtons(List(20,40,75,100),Some(40)), notAustralia),
+
+    Variant("Amount - 100 highlight Australia", "40-Australia", 1, views.html.fragments.giraffe.contributeAmountButtons(List(50, 100, 250, 500), Some(100)), Set(Australia))
```

Comment:
I don't know what the convention is for the variant name and slug I just added Australia to the end of it